src/psij/job_spec.py

```diff
@@ -132,7 +132,10 @@ def to_dict(self) -> Dict[str, Any]:
             }
             for k, v in self.attributes.__dict__.items():
                 if k in ['duration', 'queue_name', 'project_name', 'reservation_id']:
-                    d['attributes'][k] = str(v)
+                    if v:
+                        d['attributes'][k] = str(v)
```

Comment:
What's the reason for this change?

Reply:
Previous implementation did not preserve None value. The check makes sure that the value is propagated and exported correctly as null instead of "None".

Reply:
Oh, the string literal ``"None"``? That makes sense.